Makefile

```
@@ -37,7 +37,14 @@ ifdef CLI_DIR
 	OC_CLI = ${CLI_DIR}/oc
 endif
 
-CLUSTER_TYPE ?= $(shell $(OC_CLI) get infrastructures cluster -o jsonpath='{.status.platform}' | awk '{print tolower($0)}')
+ifneq ($(OC_CLI),)
+	ifeq ($(CLUSTER_TYPE),)
+		CLUSTER_TYPE := $(shell $(OC_CLI) get infrastructures cluster -o jsonpath='{.status.platform}')
+	endif
+endif
+ifneq ($(CLUSTER_TYPE),)
+	CLUSTER_TYPE := $(shell echo $(CLUSTER_TYPE) | awk "{print tolower($0)}")
```

Trying @rayfordj AWKVAR
```suggestion
	CLUSTER_TYPE := $(shell echo $(CLUSTER_TYPE) | awk -v AWKVAR="${0}" '{print tolower(AWKVAR)}')
```